cgm/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+autoreconf -fi
+./configure --prefix=$PREFIX --enable-optimize --disable-debug --enable-shared
```

Comment:
I spent ~1 hour trying to figure out how to get CGM to pickup Cubit in a preinstalled location under conda, but that was greatly challenged by the sandboxing that Conda does, and I gave up